compliance_suite/functions/client.py

```diff
@@ -24,55 +24,86 @@
 class Client():
     """ This class is used to send REST requests to the provided server URL """
 
-    def __init__(self):
-        """ Initialize the Client object"""
+    def __init__(self, service: str, server: str, version: str):
+        """ Initialize the Client object containing the server details
 
+        Args:
+            service: The API service name
+            server: The server URL to send the request
+            version: The version of the deployed API service
+        """
+
+        self.service: str = service
+        self.server: str = server
+        self.version: str = self.parse_version(version)
+        self.request_headers: dict = REQUEST_HEADERS[service]
```

Comment:
Are these guaranteed to be always the same? If not they should go back to the access methods.